src/history.cpp

```diff
@@ -1453,7 +1453,10 @@ bool history_t::save_internal_via_rewrite()
                 if (wstat(new_name, &sbuf) >= 0)
                 {
                    /* Success */
-                    fchown(out_fd, sbuf.st_uid, sbuf.st_gid);
+                    if (0 > fchown(out_fd, sbuf.st_uid, sbuf.st_gid))
```

Comment:
I would have thought to implement it that way, but the prevailing standard in the file, just below at line 1463 is
```
 if (0 > wrename(tmp_name, new_name))
```
so I mirrored that idiom.  Should I still do as you propose?  Leave it as is?  Change both?